spec/models/post_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
```

Comment:
It looks like you did not generate `Rspec` correctly. Because When I try to run tests with `rspec spec` it gives me this error: 
```
An error occurred while loading ./spec/requests/posts_spec.rb. - Did you mean?
                    rspec ./spec/requests/users_spec.rb

Failure/Error: require 'rails_helper'
```

Kindly ensure you generated the test correctly and all tests are passing. If you need guidelines on how to add RSpec to the rails app follow this [link](https://github.com/rspec/rspec-rails/tree/6-0-maintenance#:~:text=Rails%202.x.-,Installation,-IMPORTANT%20This%20README).